### ISSUE TITLE
hippo auth login returning 500

### ISSUE THREAD

**Issue by michelleN** [deislabs/hippo#589]
Tried creating a user from both the UI and the Hippo CLI (head) and I'm getting 500s trying to log in. On the UI, I was not able to create. I was able to register a user from the CLI but unable to login.

**Comment by michelleN** [comment]
I didn't realize I needed to register a user named administrator first! That worked. We should clarify in the docs.

**Comment by michelleN** [comment]
similar to https://github.com/deislabs/hippo/issues/573 but not exactly the same as I'm using the same password I used when creating the account.

**Comment by michelleN** [comment]
o0o I think I figured it out! My original password did not contain a number and a character. 
When I used `Passw0rd!`, I was able to register and login ✅ 
When I used `Password!`, I was able to register but not able to login ❌ 
When I used `Password1!`, I was able to register and login ✅ 

I believe this is just an issue of not bubbling up the password requirements. We should also be throwing an error if the password does not meet the requirements on registering a user.

**Comment by michelleN** [comment]
Looks like the server is returning the right errors but the CLI is not returning the error. From the hippo server:

```
warn: Microsoft.AspNetCore.Identity.UserManager[14]
      User password validation failed: PasswordRequiresNonAlphanumeric;PasswordRequiresDigit;PasswordRequiresUpper.
```

**Comment by bacongobbler** [comment]
I think we're either ignoring or mis-handling some exceptions raised by ASP.NET identity while creating the account. We should definitely investigate/write a few tests to capture these exceptions.

**Comment by bacongobbler** [comment]
Here's what I see if I try to register an account through the UI and the password does not meet the minimum requirements:

![image](https://user-images.githubusercontent.com/1360539/161640327-3bc6a3b9-7344-4e63-aa81-99ed45890b63.png)

Just a bunch of red bars around the password/password confirmation fields, but no information what went wrong.

And here's the CLI output:

```
><> hippo auth register -k
Enter username: foobar
Enter password: [hidden]
Error: One or more validation errors occurred. {"Password": ["The length of 'Password' must be at least 8 characters. You entered 4 characters."]}
```

I think there's room for improvement on the UI end.

**Comment by bacongobbler** [comment]
I think we fixed the majority of UX issues in #672. Tracking the UI issue in #673. Thanks @michelleN!
